PartsAppTests/ExcelHelperTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace PartsAppTests
+{
+    [TestClass]
+    public class ExcelHelperTests
+    {
+        private readonly Func<string, string> _method = PartsApp.ExcelHelper.OperationsExcelHelper.GetValidExcelBookTitle;
+
+        [TestMethod]
+        public void GetValidExcelBookTitle_TitleIsNull_ThrowsArgumentException()
+        {
+            string title = null;
+            Assert.ThrowsException<ArgumentException>(() => _method(title));
+        }
+
+        [TestMethod]
+        public void GetValidExcelBookTitle_TitleIsEmpty_ThrowsArgumentException()
+        {
+            string title = string.Empty;
+            Assert.ThrowsException<ArgumentException>(() => _method(title));
+        }
```

Comment:
The two test cases are exactly the same and differ only with 'title' that should be a parameter of the test case. you need to use [DataRow] [attribute](https://learn.microsoft.com/en-us/visualstudio/test/how-to-create-a-data-driven-unit-test?view=vs-2022#inline-data-driven-test)

Reply:
done